data/seeds/001_Parents.js

```diff
@@ -6,7 +6,9 @@ const parents = ['Alice', 'Barbara', 'Christopher', 'David'].map(
   })
 );
 
+
 exports.seed = function (knex) {
   // Inserts seed entries
   return knex('Parents').insert(parents);
+   
```

Comment:
Please remove the extra lines 9 and 13.

Reply:
@Ksouffle Why am I removing these lines?